types/svg-maps__brazil/svg-maps__brazil-tests.ts

```diff
@@ -0,0 +1,19 @@
+import * as brazil from 'svg-maps__brazil';
+
+const SVGMapContainer: brazil.Map = {
+  viewBox: 'viewBox',
+  locations: [{
+    path: 'path',
+    id: 'id',
+    name: 'name'
+  }],
+  label: 'label'
+};
+
+const SVGMapContainerNoOptional: brazil.Map = {
+  viewBox: 'vieweBox',
+  locations: [{
+  path: 'path',
+  id: 'id',
+  }],
+};
```

Comment:
```suggestion
const SVGMapContainer: brazil.Map = {
    viewBox: 'viewBox',
    locations: [
        {
            path: 'path',
            id: 'id',
            name: 'name',
        },
    ],
    label: 'label',
};

const SVGMapContainerNoOptional: brazil.Map = {
    viewBox: 'vieweBox',
    locations: [
        {
            path: 'path',
            id: 'id',
        },
    ],
};
```
just white-space align with other types (4 spaces)